CHANGELOG.md

```diff
@@ -1,5 +1,45 @@
 # AppSignal for Node.js Changelog
 
+## 3.0.0
+
+Welcome to AppSignal for Node.js version 3.0.0.!
+
+In this release, we have switched to using OpenTelemetry internally for tracing. (We do not yet support OpenTelemetry for metrics and logging.) It is no longer needed to use AppSignal instrumentation packages like `@appsignal/express`, `@appsignal/koa`, `@appsignal/nextjs`, `@appsignal/apollo-server`, etc. Some official OpenTelemetry instrumentation packages are bundled with our package as dependencies and instrumented automatically. Other _supported instrumentation packages_ can be [added and configured manually](https://docs.appsignal.com/nodejs/3.x/instrumentation/additional-instrumentations.html).
```

Comment:
I think it's good to say that we still provide support for metrics in this version.